src/MapView/MapViewTypes.ts

```diff
@@ -6,6 +6,8 @@ export type MapViewProps = {
     accessToken: string;
     // Style applied to MapView component. Note some of the View Style props are not available on ViewMap
     style: StyleProp<ViewStyle>;
+    // Focus state of the screen
+    isFocused: boolean
```

Comment:
please refer to [this comment](https://github.com/Expensify/react-native-x-maps/pull/32/files#r1306697433) 🙇